File: Common/Entity/src/main/java/it/ey/dto/AllegatoDTO.java
package it.ey.dto;

import lombok.*;
import lombok.experimental.SuperBuilder;



@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@SuperBuilder(toBuilder = true)
public class AllegatoDTO extends BaseDTO{

    private Long id;
    private Long idEntitaFK;
    private String codDocumento;
    private String codTipologiaFK;
    private String codTipologiaAllegato;
    private String descrizione;
    private String downloadUrl;
    private String sizeAllegato;
    private String type;
    private Boolean isDoc;
    private String base64;
    private LogoDTO logo;
    private String statusAllegato;
}

File: PIAO_FE/src/app/shared/models/classes/allegato-dto.ts
import { CampiTecniciDTO } from './campi-tecnici-dto';

export class AllegatoDTO extends CampiTecniciDTO {
  id?: number;
  idEntitaFK?: number;
  codDocumento?: string;
  codDocumentoFE?: string;
  codTipologiaFK?: string;
  codTipologiaAllegato?: string;
  descrizione?: string;
  downloadUrl?: string;
  sizeAllegato?: string;
  type?: string;
  isDoc?: boolean;
  base64?: string;
}
